
/NodeRgbdAlignment:
  ros__parameters:
    features:
      intensity_gradient_min: 5
      depth_gradient_min: 0.01
      depth_gradient_max: 0.3
      depth_min: 0
      depth_max: 8.0
      grid_size: 10.0
      n_levels: 4
    odometry:
      n_levels: 4
      max_iterations: 100
      min_parameter_update: 0.0001
      max_error_increase: 10
    motion_model:
      information: 10.0
    keyframe_selection:
      max_entropy_reduction: 0.05
    sync:
      queue_size_max: 50
      queue_size_min: 10
      max_interval: 0.02
    tf:
      base_link_id: world
      frame_id: odom
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
      /tf:
        publisher:
          depth: 100
          durability: volatile
          history: keep_last
          reliability: reliable
/NodeReplayer:
  ros__parameters:
    delay: 0.0
    duration: -15.0
    timeout: 0.1


